packages/csv-parser-discount-code/readme.md

```diff
@@ -0,0 +1,11 @@
+# commercetools-csv-parser-discount-code
+
+Convert [commercetools discount codes](http://dev.commercetools.com/http-api-projects-discountCodes.html) CSV data to JSON
+
+https://commercetools.github.io/nodejs/cli/csv-parser-discount-code.html
```

Comment:
This is the same template used in **all** modules in the monorepo. Should we also update all others?